narwhal/types/src/primary.rs

```diff
@@ -1941,6 +1986,13 @@ pub struct RequestVoteResponse {
     pub missing: Vec<CertificateDigest>,
 }
 
+/// Request for sending random beacon signatures to a peer.
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct SendRandomnessPartialSignaturesRequest {
+    pub round: RandomnessRound,
+    pub sigs: Vec<RandomnessPartialSignature>,
```

Comment:
Maybe switch these to bytes too to be consistent?

Reply:
I think that would harm readability without helping performance because the only place this message is used is in the handler, which requires deserialized contents